classes/hook_callbacks.php

```diff
@@ -0,0 +1,37 @@
+<?php
+// This file is part of Moodle - https://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <https://www.gnu.org/licenses/>.
+
+namespace mod_cms;
+
+/**
+ * Hook callbacks for tool_redirects.
+ *
+ * @package     mod_cms
+ * @author      Alexander Van der Bellen <alexandervanderbellen@catalyst-au.net>
+ * @copyright   2024 Catalyst IT Australia
+ * @license     https://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class hook_callbacks {
+
+    /**
+     * Listener for the after_config hook.
+     *
+     * @param \core\hook\after_config $hook
+     */
+    public static function after_config(\core\hook\after_config $hook): void {
+        // The original callback performs no operations, so we don't need to do anything here.
```

Comment:
Original hook is there to make sure vendor libs are loaded. You probably need to load then in the hook. 
See https://github.com/catalyst/moodle-tool_dataflows/blob/MOODLE_405_STABLE/classes/local/hooks/after_config.php#L39

Reply:
@Fragonite  should we merge this one first https://github.com/catalyst/moodle-mod_cms/pull/131 and then upgrade?